frame/support/test/tests/pallet.rs

```diff
@@ -262,6 +262,7 @@ pub mod pallet {
 	#[pallet::storage_prefix = "Value2"]
 	pub type RenamedValue<T> = StorageValue<Value = u64>;
 
+	/// Test some doc
```

Comment:
Can we also get a test on this so that we know the metadata generated for it is correct?

Reply:
actually I don't know how to test it.
I tested it manually, I think it is enough

Reply:
You should be able to get the metadata for this storage item? And then check that the comment is present?

Reply:
if I not mistaken type_value doesn't go into metadata